docs/4-Commands/kool-logs.md

```diff
@@ -1,6 +1,12 @@
 ## kool logs
 
-Display log output from all or a specific service container
+Display log output from running service containers
+
+### Synopsis
+
+Display log output from all running service containers,
+or one or more specified [SERVICE...] containers. Add a '-f' flag to the
+end of the command to follow the log output (i.e. 'kool logs [SERVICE...] -f').
```

Comment:
@dbpolito @fabriciojs - FYI - I added this last sentence about the `-f` flag for `kool logs`, so please give this change extra attention.

Reply:
Well the `-f` doesn't necessarily need to be after service... maybe for consistency we can follow the `[OPTIONS]` patter and be fore the arg... so it could be `kool logs -f service` or just `kool logs -f` which will follow all

Reply:
@dbpolito - you're absolutely right. Let me quickly fix and update my PR.

Reply:
@dbpolito - updated